### PR TITLE
Xeno caste changes, mostly buffs.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Changes the maturity scaling of most benos. This is mostly the same as #3774 until Kuro orphaned it. So I had to do it.
 ## Why It's Good For The Game
Smooths the curve out for most benos. It's helpful for curbing the powertide.

## Changelog
:cl:

balance: A lot of beno changes. Too many to list. Check out the PR! Checkout https://github.com/tgstation/TerraGov-Marine-Corps/pull/3831 !
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm [hunk 1]
```diff
@@ -141,23 +141,23 @@
...
 	plasma_max = 1000
 	plasma_gain = 50
 
 	// *** Health *** //
-	max_health = 300
+	max_health = 275
 
 	// *** Defense *** //
-	armor = list("melee" = 37, "bullet" = 37, "laser" = 37, "energy" = 37, "bomb" = XENO_BOMB_RESIST_0, "bio" = 37, "rad" = 37, "fire" = 25, "acid" = 37)
+	armor = list("melee" = 42, "bullet" = 42", "laser" = 42, "energy" = 42, "bomb" = XENO_BOMB_RESIST_0, "bio" = 37, "rad" = 37, "fire" = 25, "acid" = 37)
```
Comment (psykzz) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm:160]:
```
code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm, line 160, column 42:
error: got '", "', expected one of: operator, field access, ')', ','
- 161:1: check for extra indentation at the start of the next line
- fatal error: the parser cannot continue
- constant evaluation will be skipped
```

Comment (psykzz) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm:160]:
```suggestion
	armor = list("melee" = 42, "bullet" = 42, "laser" = 42, "energy" = 42, "bomb" = XENO_BOMB_RESIST_0, "bio" = 37, "rad" = 37, "fire" = 25, "acid" = 37)
```

**Comment by drawsstuff** [comment]
The hugger delay is a bit much. Carriers need a way to defend themselves, and putting a 3 second timer on the panic button doesnt help.

**Comment by MerryweatherLost** [comment]
**Update**
Going to still assume that code-wise it's fine. We will check it out next test to see the balance ratio.

**Comment by AdamElTablawy** [comment]
these are in no way 'mostly buffs'

**Comment by Rohesie** [comment]
Give a bit more detailed feedback, Adam.

**Comment by AdamElTablawy** [comment]
@Rohesie 

look at the actual changes

they're all nerfs

slower, less damage, longer cooldowns, etc

pr title is a straight up lie, no buffs

**Comment by psykzz** [comment]
@AdamElTablawy lower speed = faster xenos.
Health has gone down, but ii see armour increases and damage increases as well.

Overall it looks more like a rebalance vs a nerf at all.

**Comment by AdamElTablawy** [comment]
@psykzz Yes, I know how the speed var works.

You've slowed them down, not sped them up.

**Pull request merged**